src/Controller/ApplePay.php

```diff
@@ -459,6 +459,12 @@ public function process_standard_payment($order, $amount, $customer_id, $workflo
 				]
 			];
 
+			if (PayplugWoocommerceHelper::is_checkout_block()) {
```

Comment:
this methods `is_checkout_block` && `is_cart_block` only says if you've checkout blocks activated or not, not if the payment was done on the cart page or checkout page. please check this